content/post/Android_Root101.en.md

```diff
@@ -49,16 +50,18 @@ Place this file inside the folder created after extraction.
 > `cd..` is used to go down one directory
```

Comment:
Can't comment on the line above, but 
`is used to go to the directory with the same name`  or something
will be better imo
going up the directory is inaccurate, the file system not linear